apps.awesim.org/apps/myjobs/templates/Basic_LAMMPS_Parallel_Job_Pitzer-Exp/slurm_script.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#SBATCH -J ondemand/sys/myjobs/basic_lammps_parallel
+#SBATCH --time=00:30:00
+#SBATCH --nodes=2
+#SBATCH --ntasks-per-node=48
```

Comment:
This wil lmake the job only runable on Pitzer expansion, I would instead advise removing `--ntasks-per-node` and as SLURM jobs >= 2 nodes will be whole-node jobs at OSC